Source/VisualStudioTools/Private/VisualStudioToolsCommandlet.cpp

```diff
@@ -380,6 +381,29 @@ static void RunAssetScan(
 			FObjectPropertyBase::GetExportPath(Class.Get(), nullptr /*Parent*/, nullptr /*ExportRootScope*/, 0 /*PortFlags*/));
 		});
 
+	// Take account of any core redirects for the blueprint classes we want to scan.
+	auto FilterArray = Filter.TagsAndValues.Array();
+	for (auto OneEntry : FilterArray)
+	{
+		if (OneEntry.Value.IsSet())
+		{
+			// I don't know what this CoreUObject wrapper is, but it needs removing and then adding back to any previous name.
+			FString ClassName = OneEntry.Value.GetValue();
+			ClassName.RemoveFromStart("/Script/CoreUObject.Class'");
+			ClassName.RemoveFromEnd("'");
+
+			TArray<FCoreRedirectObjectName> PreviousNames;
+			if (FCoreRedirects::FindPreviousNames(ECoreRedirectFlags::Type_Class, ClassName, PreviousNames))
+			{
+				for (auto PreviousName : PreviousNames)
```

Comment:
Small detail, but this should be `for (auto&` instead, because `auto` does not bind to references, so this loop causes a copy operation for each element of the array.

Reply:
Good spot thanks :)
Submitted this change.